orocos_kdl/src/treeiksolverpos_online.hpp

```diff
@@ -61,7 +61,7 @@ class TreeIkSolverPos_Online: public TreeIkSolverPos {
      * @return
      */
 
-    TreeIkSolverPos_Online(const double& nr_of_jnts,
+    TreeIkSolverPos_Online(const unsigned int& nr_of_jnts,
```

Comment:
This is an API breaking change. This has been discussed in https://github.com/orocos/orocos_kinematics_dynamics/issues/384. So I think we should skip this one.

Reply:
Updated to avoid breaking API: [d70e705](https://github.com/orocos/orocos_kinematics_dynamics/pull/390/commits/d70e705f2f766adc4b22f931a3aaa7ffd470feac)

Reply:
Not sure why, but it looks like this change ([d70e705](https://github.com/orocos/orocos_kinematics_dynamics/pull/390/commits/d70e705f2f766adc4b22f931a3aaa7ffd470feac)) broke a test :thinking:

Reply:
Since most of the members we were initializing with this value where being overridden immediately in the constructor, I've just removed them from the initializer list ([fc0dfd8](https://github.com/orocos/orocos_kinematics_dynamics/pull/390/commits/fc0dfd80cbe1810f7e27c10a860c7c1896174f99)). CI looks happy now.